backend/src/migrations/20240829211847_add_notifications_table.ts

```diff
@@ -0,0 +1,49 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'notifications';
+
+const ColumnName = {
+    ID: 'id',
+    USER_ID: 'user_id',
+    TYPE: 'type',
+    IS_READ: 'is_read',
+    CREATED_AT: 'created_at',
+    UPDATED_AT: 'updated_at',
+};
+
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";');
+    await knex.schema.createTable(TABLE_NAME, (table) => {
+        table
+            .uuid(ColumnName.ID)
+            .notNullable()
+            .primary()
+            .defaultTo(knex.raw('uuid_generate_v4()'));
+        table.uuid(ColumnName.USER_ID).notNullable();
+        table
+            .enu(ColumnName.TYPE, ['render'], {
+                useNative: true,
+                enumName: 'notification_type',
+            })
+            .notNullable();
+        table.boolean(ColumnName.IS_READ).notNullable();
```

Comment:
You could create a default false value to is read. 
table.boolean(ColumnName.IS_READ).notNullable().defaultTo(false);

Reply:
I think its a good idea

Reply:
I think its a good idea

Reply:
Done